modules/data-loading/pages/data-streaming-connector/index.adoc

```diff
@@ -108,7 +108,151 @@ Use the `-s` or `--soft` flag to delete the connector only and leave the Kafka m
 $ gadmin connector delete <connector_name> -s
 ----
 
+== Create the loading job
 
+The loading job creation happens entirely in GSQL.
+
+=== Specify a graph and config file
+
+[source.wrap, gsql]
+----
+USE GRAPH test_graph
+CREATE DATA_SOURCE STREAM s1 = "ds_config.json" for graph test_graph
+----
+
+In this example, we create a data source of type `STREAM` and name it `s1`.
+We set our data source as the `ds_config.json` file.
```

Comment:
ds_config.json is not the data source; it is the steam's config file.
That confusion is the fault of the command syntax.  The syntax should not be 
`STREAM s1 = <config_file>`
An example of a semantically correct syntax is
`STREAM s1 WITH CONFIG <config_file>`